app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  skip_before_action :verify_authenticity_token
```

Comment:
We could add a `before_action :authenticate_user!`. Then, I don't think that we need to skip this action, do you?

Source: https://devise-token-auth.gitbook.io/devise-token-auth/usage/controller_methods#methods